graphium/data/datamodule.py

```diff
@@ -970,30 +996,51 @@ def prepare_data(self, save_smiles_and_ids: bool = False):
         - In the previous step, we were also able to get the unique smiles, which we use to compute the features
         - For each single-task dataframe and associated data (smiles, labels, etc.):
             - Filter out the data corresponding to molecules which failed featurization.
-            - Create a corresponding SingletaskDataset
-            - Split the SingletaskDataset according to the task-specific splits for train, val and test
+            - Split the dataset according to the task-specific splits for train, val and test
         """
 
-        def has_atoms_after_h_removal(smiles):
-            # Remove all 'H' characters from the SMILES
-            smiles_without_h = re.sub("H", "", smiles)
-            # Check if any letters are remaining in the modified string
-            has_atoms = bool(re.search("[a-zA-Z]", smiles_without_h))
-            if has_atoms == False:
-                logger.info(f"Removed Hydrogen molecule: {smiles}")
-            return has_atoms
+        # Don't log error messages from SMILES parsing in RDKit.
+        # Common error messages were:
+        # WARNING: not removing hydrogen atom without neighbors
+        # SMILES Parse Error: syntax error while parsing: restricted
+        # SMILES Parse Error: Failed parsing SMILES 'restricted' for input: 'restricted'
+        RDLogger.DisableLog('rdApp.*')
```

Comment:
I'm okay either way.  I erred on the side of stopping the logging, because the conditions are safely handled by the code that calls RDKit, and the way RDKit does the logging repeatedly breaks the training progress bar.  I think it was a few dozen times per epoch in LargeMix.

Reply:
okok, it's fine to keep it here then.